### DISCUSSION TITLE
Teardown after all tests have run

### DISCUSSION THREAD

**Discussion by drFrankWhite** [seleniumbase/SeleniumBase#2326] [Q&A] [answered]
Each of my test classes has its own teardown, but how do I write a teardown that runs once all tests finish?  Similar to Pytest's `scope=module`, or maybe `session`

**Comment by mdmintz** [comment, marked as answer]
To have a special tearDown after all tests have run, override `tearDownClass()`, similar to how [Syntax Format 2](https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/syntax_formats.md#sb_sf_02) overrides `tearDown()` in [SeleniumBase/examples/boilerplates/base_test_case.py](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/boilerplates/base_test_case.py)

Looks like someone did something like that here: https://github.com/seleniumbase/SeleniumBase/issues/317#issuecomment-1129870556
